utilcode/src/main/ets/utils/ArrayUtils.ets

```diff
@@ -0,0 +1,295 @@
+/*
+ * Copyright (c) 2022 Application Library Engineering Group.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export class ArrayUtils {
+  constructor() {
+  }
+
+  static readonly INDEX_NOT_FOUND = -1;
+
+  static newArray<Type>(array: Type[]): Type[] {
+    return <Type[]>array;
+  }
+
+  static isEmpty(array) {
+    if (array)
+    return array.length == 0;
+    else
+    return true;
+  }
+
+  static getLength(array) {
+    if (array)
+    return array.length;
```

Comment:
format all the ets files, open each file and perform ctrl + Alt + L to format the code.